lib/net.js

```diff
@@ -578,7 +578,7 @@ Socket.prototype._destroy = function(exception, cb) {
   if (this._handle) {
     if (this !== process.stderr)
       debug('close handle');
-    var isException = exception ? true : false;
+    const isException = !!exception;
```

Comment:
Ok with the `var` -> `const` change but any reason for the double negation? It seems only churn to me.

Reply:
convert to boolean. and other places is `!!xxx` too